isolate_example/lib/page_one.dart

```diff
@@ -68,11 +154,17 @@ class SmoothAnimationWidgetState extends State<SmoothAnimationWidget>
     _controller.forward();
   }
 
+  @override
```

Comment:
Having the dispose function at the end of this class might be better, so reading the reading experience is more chronological.